nengo_viz/static/viz_value.js

```diff
@@ -27,8 +27,10 @@ VIZ.Value = function(parent, sim, args) {
     /** scales for mapping x and y values to pixels */
     this.scale_x = d3.scale.linear();
     this.scale_y = d3.scale.linear();
-    this.scale_y.domain([args.miny || -1, args.maxy || 1]);
-    
+    var miny = (args.miny !== null) ? args.miny : -1;
+    var maxy = (args.maxy !== null) ? args.maxy : 1;
```

Comment:
Yeah, I was wondering about getting rid of it. Python's `None` seems to get mapped to `null` in JavaScript, which is why I used it.